torchgeo/datasets/advance.py

```diff
@@ -0,0 +1,229 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""ADVANCE dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, List, Optional
+
+import numpy as np
+import torch
+import torchaudio
```

Comment:
If you want this dependency to be optional, it needs to be a lazy import inside of the function that uses it. Otherwise, this will always get imported any time you import anything from `torchgeo.datasets`.

Reply:
Removed torchaudio dependency.